portfolio/src/main/webapp/style.css

```diff
@@ -5,9 +5,31 @@
   padding: 50px;
   margin: 20px;
   position: relative;
-  left:20%;
+  left: 20%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* Create and customize a navigation bar at
```

Comment:
I guess it doesn't exceed 80 characters. Why do you split one sentence into two lines?

Reply:
Fixed!